src/index.js

```diff
@@ -60,8 +60,74 @@ export function renderToString(vnode, context) {
 			context || EMPTY_OBJ,
 			false,
 			undefined,
-			parent
+			parent,
+			false
 		);
+	} catch (e) {
+		if (e.then) {
+			throw new Error('Use "renderToStringAsync" for suspenseful rendering.');
+		}
```

Comment:
Only just noticed, this is breaking for all preact-iso users... do we want to downgrade this to a warning message instead of throwing an error, or just accept the breaking change in a minor?